dashboard/app/jobs/concerns/active_job_metrics.rb

```diff
@@ -31,18 +31,27 @@ module ActiveJobMetrics
   end
 
   protected def report_job_count
-    Cdo::Metrics.push(
-      METRICS_NAMESPACE, [
+    Cdo::Metrics.push(METRICS_NAMESPACE,
+      # Same metrics as "bin/cron/report_activejob_metrics"
+      [
         {
-          # Same metric as "bin/cron/report_activejob_metrics"
-          metric_name: 'JobCount',
-          value: Delayed::Job.count,
+          metric_name: 'PendingJobCount',
+          value: Delayed::Job.where(failed_at: nil).count,
           unit: 'Count',
           timestamp: Time.now,
           dimensions: [
             {name: 'Environment', value: CDO.rack_env},
           ],
         },
+        {
+          metric_name: 'FailedJobCount',
+          value: Delayed::Job.where.not(failed_at: nil).count,
```

Comment:
ohh, I see now why you want to stub the `Delayed::Job.where` call, so that you can get the right value here. one idea is to introduce a helper method
```
def get_failed_jobs
  Delayed::Job.where.not(failed_at: nil).count
end
```
and just stub that method instead.

Reply:
That works! I added helper methods and a quick test for each. That should be making it's way through Drone now.